OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj

```diff
@@ -23,6 +23,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="Eluant">
+      <HintPath>../engine/thirdparty/download/Eluant.dll</HintPath>

```

Comment:
What is this used for?

Reply:
> The type 'ILuaToStringBinding' is defined in an assembly that is not referenced. You must add a reference to assembly 'Eluant, Version=1.0.5867.35197, Culture=neutral, PublicKeyToken=null'. [OpenRA.Mods.RA2]

In `BaseSpawnerParent` line 227. Must be an indirect reference.

Reply:
It can be removed once the engine matches https://github.com/OpenRA/OpenRA/pull/17744/.